.github/workflows/CI.yml

```diff
@@ -114,10 +114,3 @@ jobs:
         run: |
           VALGRIND_CMD="pyvrp pyvrp/tests/data/RC208.txt --instance_format solomon --round_func dimacs --max_runtime 0.5 --seed 1"
           poetry run env PYTHONMALLOC=malloc valgrind --tool=memcheck --quiet $VALGRIND_CMD
-      - if: matrix.compiler == 'clang' && matrix.python-version == '3.11'
-        # Run this only for one job to reduce workflow runtimes.
-        name: Run example notebooks
-        run: |
-          for f in examples/*.ipynb;
-            do poetry run jupyter nbconvert --to notebook --execute $f;
-          done
```

Comment:
RTD now executes each notebook before rendering the docs, so this is not needed anymore.